src/geometry/euclidean/basic.lean

```diff
@@ -578,6 +579,128 @@ lemma angle_right_midpoint_eq_pi_div_two_of_dist_eq {p1 p2 p3 : P} (h : dist p3
   ∠ p3 (midpoint ℝ p1 p2) p2 = π / 2 :=
 by rw [midpoint_comm p1 p2, angle_left_midpoint_eq_pi_div_two_of_dist_eq h.symm]
 
+/-- If the second of three points is strictly between the other two, the angle at that point
+is π. -/
+lemma _root_.sbtw.angle₁₂₃_eq_pi {p₁ p₂ p₃ : P} (h : sbtw ℝ p₁ p₂ p₃) : ∠ p₁ p₂ p₃ = π :=
```

Comment:
```suggestion
lemma _root_.sbtw.angle_eq_pi {p₁ p₂ p₃ : P} (h : sbtw ℝ p₁ p₂ p₃) : ∠ p₁ p₂ p₃ = π :=
```
There might be a local naming scheme I'm not aware of, but this name seems better (and `sbtw.angle_rev_eq_pi` or `sbtw.angle_swap_eq_pi` for the next)

Reply:
Looking at your later lemmas, this naming scheme makes sense. You could turn `angle₂₁₃` into `angle_left` and `angle₃₁₂` into `angle_left_swap`, but your naming scheme might be better.

Reply:
Yes, I wanted something consistent for all six `sbtw` lemmas, plus the `wbtw` variants, and using subscripts was the best thing I could come up with for naming such a family of lemmas for six permutations.